docs/DEPLOYMENT.md

```diff
@@ -116,13 +116,12 @@ server {
 sudo certbot --nginx
 ```
 
-+ **Running** drand now requires to add the following options:
++ **Running** drand uses two ports: one for group member communication, and one for a public-facing API for distributing randomness. These ports, and interfaces should be specified with flags.
```

Comment:
I would add that the public-facing API is exposing a HTTP endpoint that exposes a restricted set of methods (potentially link to ...somewhere?) and the private-facing one is the regular gRPC endpoint use to talk between drand nodes.

Reply:
Cloudflare's not treating both the same way so lets make it clear what each means.